fingertip/plugins/cleanup.py

```diff
@@ -100,6 +101,39 @@ def tempfiles(older_than='4h', location=None):
                                       temp.AUTOREMOVE_PREFIX not in f))
 
 
+def saviour(expired_for='10d'):
+    if expired_for != 'all':
+        adjusted_time = time.time() - units.parse_time_interval(expired_for)
+    else:
+        adjusted_time = 0
+    root, dirs, files = next(os.walk(path.SAVIOUR + '/_'))
```

Comment:
One other thing to clean up is the non-`_` directories. I hope there the logic is gonna be simpler, just remove everything that's a broken symlink.